Client/mods/deathmatch/logic/luadefs/CLuaPedDefs.cpp

```diff
@@ -61,6 +62,7 @@ void CLuaPedDefs::LoadFunctions()
         {"setPedEnterVehicle", ArgumentParser<SetPedEnterVehicle>},
         {"setPedExitVehicle", ArgumentParser<SetPedExitVehicle>},
         {"setPedBleeding", ArgumentParser<SetPedBleeding>},
+        {"pedSay", ArgumentParser<PedSay>},
```

Comment:
Personally, I would rename this to `playPedVoiceLine`. What do you think?

Reply:
Good idea